index.html

```diff
@@ -55,8 +52,8 @@
             <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12 hero-text">
                 <p class="hero_desc">Hi,</p>
                 <p class="hero_desc">Welcome to My Portfolio Website!</p>
-                <p class="hero_desc I'm">I'm</p>
```

Comment:
Great job with this project :muscle:
Could you kindly consider changing the class of I'm to something else? I will also suggest you avoid using apostrophes in class names.